op-chain-ops/deployer/pipeline/superchain.go

```diff
@@ -40,7 +40,7 @@ func DeploySuperchain(ctx context.Context, env *Env, artifactsFS foundry.StatDir
 				dso, err = opcm.DeploySuperchain(
 					host,
 					opcm.DeploySuperchainInput{
-						ProxyAdminOwner:            intent.SuperchainRoles.ProxyAdminOwner,
+						SuperchainProxyAdminOwner:  intent.SuperchainRoles.ProxyAdminOwner,
```

Comment:
We should change this in the intent also

Reply:
I wasn't sure so I didn't. I can change though if we want.

Reply:
My reasoning was that the `DeploySuperchainInput` was descriptive enough such that the intent didn't need to be. We don't lose anything adding it to the intent so I'll make that change.

Reply:
https://github.com/ethereum-optimism/optimism/pull/12106/commits/19257da7b70b2bee848f41f331e4bd23ccf033bc

Reply:
Thanks. I don't feel strongly, so will defer to you and @mslipper here on preferences

Reply:
i leave as-is in the intent - seems weird to prefix everything in a data structure called "superchain roles" with "superchain"

Reply:
https://github.com/ethereum-optimism/optimism/pull/12106/commits/650a5e44473ea1482525be015218094df580cb5e